server/imports/core/connection/index.js

```diff
@@ -1,77 +1,154 @@
+/* eslint-disable nonblock-statement-body-position */
 import { Database, Logger, Error } from '/server/imports/modules';
 import ConnectionHelper from './helper';
 
 const fs = require('fs');
 const mongodbUrlParser = require('mongodb-url');
 const jsonParser = require('json5');
 
-const Connection = function () {
-};
+const Connection = function () {};
 
 const addSSLOptions = function (obj, result) {
   if (obj.rootCAFile) {
     result.sslValidate = true;
     result.sslCA = Buffer.from(obj.rootCAFile);
   }
   if (obj.certificateFile) result.sslCert = Buffer.from(obj.certificateFile);
-  if (obj.certificateKeyFile) result.sslKey = Buffer.from(obj.certificateKeyFile);
+  if (obj.certificateKeyFile) {
+    result.sslKey = Buffer.from(obj.certificateKeyFile);
+  }
   if (obj.passPhrase) result.sslPass = obj.passPhrase;
   if (obj.disableHostnameVerification) result.checkServerIdentity = false;
 };
 
 const setOptionsToConnectionFromParsedUrl = function (connection, parsedUrl) {
   if (parsedUrl.server_options) {
-    connection.options.connectionTimeout = (parsedUrl.server_options.socketOptions && parsedUrl.server_options.socketOptions.connectTimeoutMS)
-      ? parsedUrl.server_options.socketOptions.connectTimeoutMS : '';
-    connection.options.socketTimeout = (parsedUrl.server_options.socketOptions && parsedUrl.server_options.socketOptions.socketTimeoutMS)
-      ? parsedUrl.server_options.socketOptions.socketTimeoutMS : '';
+    connection.options.connectionTimeout =
```

Comment:
Codacy has a fix for the issue: [There should be no line break before or after '='.](https://app.codacy.com/gh/nosqlclient/nosqlclient/pullRequest?prid=7730964)
```suggestion
    connection.options.connectionTimeout =      parsedUrl.server_options.socketOptions &&
```